client/apps/edit/components/admin/components/verticals_tags.jsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import { data as sd } from 'sharify'
+import Verticals from '../../../../../collections/verticals.coffee'
+import { AutocompleteInlineList } from '/client/components/autocomplete2/inline_list'
+
+export class AdminVerticalsTags extends Component {
+  static propTypes = {
+    article: PropTypes.object,
+    onChange: PropTypes.func
+  }
+
+  constructor (props) {
+    super(props)
+    const { article } = props
+
+    this.state = {
+      vertical: article.get('vertical') || null,
+      verticals: []
+    }
+  }
+
+  componentWillMount = () => {
+    this.fetchVerticals()
+  }
+
+  fetchVerticals = () => {
+    new Verticals().fetch({
+      cache: true,
+      success: (verticals) => {
+        let sortedVerticals = verticals.sortBy('name')
```

Comment:
Any reason this has to be a `let`? It doesn't look like we're reassigning it elsewhere. Generally I try to stick with `const` unless necessary.